kDriveCore/Data/Upload/UploadQueue/Queue/UploadQueue.swift

```
@@ -34,6 +34,9 @@ public class UploadQueue: ParallelismHeuristicDelegate {
 
     private var queueObserver: UploadQueueObserver?
 
+    static let silentErrors: [DriveError] =
```

Old silent errors  + `uploadNotTerminated` and `uploadNotTerminatedError` are now silent error.

`uploadNotTerminated` and `uploadNotTerminatedError` are errors I encountered only on the direct upload path.
Both pre-existed in the code base, as it predates session upload. But a session upload will not trigger them in my testing.